daemon/migrations/20211025050345_rename_term_to_settlement_time_interval.sql

```diff
@@ -1,8 +1,59 @@
-alter table orders
-rename column term_seconds to settlement_time_interval_seconds;
+PRAGMA foreign_keys=off;
```

Comment:
You should never change old migration sql files. 
These are meant to be incremental. 

Here a short how to use sqlx-cli: 
https://github.com/launchbadge/sqlx/blob/master/sqlx-cli/README.md#sqlx-cli

If you look at the filename it says : 

`20211025050345_rename_term_to_settlement_time_interval.sql`

So on `2021 - 11 - 02` we renamed the term `settlement time` to `interval`. 

You changed the content to something completely different and the filename does not fit anymore

Differently said, any change to the db should have its own migration file. 

This way you can track how the database has evolved over time.

Reply:
Note that the `sqlx.json` file is unaffected by this, meaning that I am not altering the incremental revision approach at all but merely changing the syntax used to be something that is more "correct" in the sense of it not requiring us to be in a limited-case scenario, which is what was done previously.

The outcomes are identical in both cases.

Reply:
Thanks for pointing this out. 

Regarding the changes: it's not clear why this is necessary. In your commit message you mention: 

> Not completely necessary, but the previous ALTER commands only worked
because of being operated in an edge-case.

What are the edge-cases and when could the old approach not work? 

The old file was much smaller and imho easier to understand.

Reply:
Please refer to the SQLite docs for a complete answer; short version is that the `ALTER` command only worked because of there being no fkeys involved, and **possibly** (not sure about this) being so early in the game. In other words, it is implicitly working on an edge-case where it manages to work by accident, not design. This edit makes things clearer going forward regarding the use of the recommended process is all.

As I have stated already though, not completely necessary at all.

Reply:
I don't understand what you are saying :( 

> short version is that the ALTER command only worked because of there being no fkeys involved, 

Differently said, it works if no foreign keys are involved. Which there aren't, so that's fine then? 
Since schema migration files are run sequentially, this fact will never change, i.e. i.e. the columns being altered here will never become foreign keys. Differently said, if it works now, it will always work. Or am I misunderstanding something? 

> and possibly (not sure about this) being so early in the game

What do you mean with this?

Maybe you can point me to that part of the SQLite docs you are referring to please?

Reply:
Since this PR doesn't actually "do" anything (it's for syntax purposes only) I propose this simply gets closed as it is clearly causing too much hassle.